### ISSUE TITLE
Error Code: -7

### ISSUE THREAD

**Issue by juang0nzal3z** [ablab/spades#154]
I am assembling a two genomes independently with the following script:

spades.py -k 21,33,55,77,99,127 --careful -1 trim_SCLB_R1_paired.fq.gz  -2 trim_SCLB_R2_paired.fq.gz -o spades_assembly

with the following job:
#SBATCH --nodes=1                    # Use one node
#SBATCH --ntasks=1                   # Run a single task        
#SBATCH --cpus-per-task=32            # Number of CPU cores per task
#SBATCH --mem=64gb                    # Total memory limit         
#SBATCH --time=96:00:00    # Time limit hrs:min:sec

I get the following error message for both jobs:
== Error ==  system call for: "['/apps/spades/3.12.0/bin/spades-hammer', 'spades_assembly/corrected/configs/config.info']" finished abnormally, err code: -7

[1_params.txt](https://github.com/ablab/spades/files/2264802/1_params.txt)
[1_spades.log](https://github.com/ablab/spades/files/2264803/1_spades.log)
[2_params.txt](https://github.com/ablab/spades/files/2264804/2_params.txt)
[2_spades.log](https://github.com/ablab/spades/files/2264805/2_spades.log)

**Comment by asl** [comment]
This looks like a memory issue. Does the problem reproduce with the restart or increased memory limit?

**Comment by juang0nzal3z** [comment]
Yes, I increased the memory from 32 to 64 gb and increased the number of threads to 32. The problem is the same and consistent for two different assemblies.
